packages/synapse-react-client/src/components/Evaluation/EvaluationRoundEditor.tsx

```diff
@@ -312,8 +304,10 @@ export const EvaluationRoundEditor: React.FunctionComponent<
             <Row className="mb-3">
               <Col>
                 <Button
-                  variant="link"
-                  className="advanced-limits-link font-weight-bold"
+                  variant="text"
+                  color="primary"
+                  sx={{ fontSize: '16px !important' }}
```

Comment:
Prevent bootstrap-4-backport from overriding the font size -- 

main | feature
:---:|:---:
<img width="1176" alt="main_EvaluationRoundEditor_Buttons" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/ac80b4dc-6fa2-464f-acfe-5f56ac50dea9"> | <img width="1175" alt="feature_EvaluationRoundEditor_Buttons" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/586b4be0-da8d-41da-86d6-793370aa81a9">